src/window/advisor/ratings.c

```diff
@@ -112,9 +124,27 @@ static int draw_background(void)
         case SELECTED_RATING_CULTURE:
             lang_text_draw(53, 1, 72, 359, FONT_NORMAL_WHITE);
             if (culture <= 90) {
-                lang_text_draw_multiline(53, 9 + city_rating_selected_explanation(),
-                    72, 374, 496, FONT_NORMAL_WHITE);
-            } else {
+             int min_rating = 100;
+				int advice_text = 3;
+				if (city_data.ratings.culture_points.average_education < min_rating) {
+					min_rating = city_data.ratings.culture_points.average_education;
+					advice_text = 2;
+				}
+				if (city_data.ratings.culture_points.average_religion < min_rating) {
+					min_rating = city_data.ratings.culture_points.average_religion;
+					advice_text = 1;
+				}
+				if (city_data.ratings.culture_points.average_entertainment < min_rating) {
+					min_rating = city_data.ratings.culture_points.average_entertainment;
+					advice_text = 0;
+				} 				
+				if (city_entertainment_venue_needing_shows() > 0 ) {
+					if (city_data.ratings.culture / city_entertainment_venue_needing_shows() < 11 ) {
+						advice_text = 4;
+					}
+				}				
+				text_draw(translation_for(culture_reasons[advice_text]), 72, 374, FONT_NORMAL_WHITE, 0);
```

Comment:
You could use something like `translation_for(TR_ADVISOR_CULTURE_REASON_1 + reason_text_offset)` instead, it's what we do with image offsets anyway.